recipes/apache-maven/3.6.3/conanfile.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class PackageApacheMaven(ConanFile):
+    name = "apache-maven"
+    version = "3.6.3"
```

Comment:
Don't hardcode the version here

```suggestion
```

As a consequence also rename the folder name from `3.6.3` to `all`

Making individual folders/recipes is only needed when the recipe differs strongly from version to version